.env.dev

```diff
@@ -4,10 +4,12 @@ JWT_SECRET="thisismysupersecrettokenjustkidding"
 DATABASE_URL="mongodb://mongo:27017/donut-development"
 SENDGRID_API_KEY='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'
 SOCKET_PORT=8810
-clientbaseurl = "http://localhost:3000"
+clientbaseurl="http://localhost:3000"
 SOCKET_PORT=8810
 REDIS_HOST="redis"
 REDIS_PORT=6379
 REDIS_PASSWORD="auth"
 REDIS_DB=0
 REDIS_ACTIVITY_DB=1
+GITHUB_OAUTH_APP_CLIENTID="a3e08516c35fe7e83f43"
+GITHUB_OAUTH_APP_CLIENTSECRET="8393d95c3bfeeb0943045f447a9d055cb660bce0"
```

Comment:
whose token is this is it supposed to be present here?

Reply:
The organization admin who is setting up donut will have to setup an oauth application on GitHub, these are for that oauth application, he will have to modify these in the code he has cloned. I have setup an oauth app on GitHub and provided it's clientid and secret for testing so that it does not need to setup by every contributor who wants to try it out.